python/llm/example/transformers/transformers_int4/chatglm/README.md

```diff
@@ -19,20 +19,11 @@ After installing conda, create a Python environment for BigDL-LLM:
 conda create -n llm python=3.9 # recommend to use Python 3.9
 conda activate llm
 
-pip install bigdl-llm[all] # install bigdl-llm with 'all' option
+pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option
 ```
 
 ### 2. Run
-```
-python ./generate.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --prompt PROMPT --n-predict N_PREDICT
-```
-
-Arguments info:
-- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the ChatGLM model to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'THUDM/chatglm-6b'`.
-- `--prompt PROMPT`: argument defining the prompt to be infered (with integrated prompt format for chat). It is default to be `'AI是什么？'`.
-- `--n-predict N_PREDICT`: argument defining the max number of tokens to predict. It is default to be `32`.
-
-The expected output can be found in [Sample Output](#23-sample-output) section.
+After setting up the Python environment, you could run the example [generate.py](./generate.py) by following steps.
```

Comment:
remove generate.py here? It is linked above, too many places would be hard to maintain